src/main/java/uk/gov/justice/digital/hmpps/csr/api/controllers/DetailController.kt

```diff
@@ -1,20 +1,28 @@
 package uk.gov.justice.digital.hmpps.csr.api.controllers
 
 import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.media.Schema
 import io.swagger.v3.oas.annotations.tags.Tag
 import org.springframework.format.annotation.DateTimeFormat
 import org.springframework.http.MediaType.APPLICATION_JSON_VALUE
 import org.springframework.http.ResponseEntity
+import org.springframework.security.access.prepost.PreAuthorize
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RequestParam
 import org.springframework.web.bind.annotation.RestController
 import uk.gov.justice.digital.hmpps.csr.api.dto.DetailDto
 import uk.gov.justice.digital.hmpps.csr.api.service.DetailService
 import java.time.LocalDate
 
-@Tag(name = "details", description = "Get details of shifts from CSR")
+@Tag(
+  name = "details",
+  description = "Details of shifts from CSR. NOTE: All of these endpoints use the header value 'X-Region' to select the required region"
```

Comment:
would be better to add a header parameter to the relevant methods
i.e. something like
```
		@Parameter(in = ParameterIn.HEADER, name = "X-Region")

```

Reply:
👍